src/config.ts

```diff
@@ -3006,6 +3012,48 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://www.mintscan.io/fetchai/txs/{txHash}',
 	},
+    {
```

Comment:
Although the ordering doesn't really matter here, it'd be easier to find if this were at the end of the list. Can we move it down?

Reply:
Yes we can. I've move to the end.